apps/recnet-api/buildspec.yml

```diff
@@ -5,7 +5,7 @@ phases:
     runtime-versions:
       nodejs: 18
     commands:
-      - npm i -g pnpm
+      - npm i -g pnpm@8
```

Comment:
Please place the related issue here or in the PR about why we want to fix the version to 8. Maybe it would help future debugging.